Core/src/RootExt.cpp

```diff
@@ -103,9 +102,10 @@ void RebinAndFill(TH2& new_hist, const TH2& old_hist)
         const int bin_low_new = new_axis->FindFixBin(old_low_edge);
         const int bin_up_new = new_axis->FindFixBin(old_up_edge);
 
+        const double new_up_edge = new_axis->GetBinUpEdge(bin_low_new);
```

Comment:
I guess here  `new_axis->GetBinUpEdge(bin_low_new)` is needed not `old_axis->GetBinUpEdge(bin_low_new)`